pkg/resources/resource_monitor.go

```diff
@@ -283,13 +277,54 @@ func extractTriggerInts(s sql.NullString) ([]int, error) {
 	return out, nil
 }
 
+// inSlice returns true if n is in string slice h otherwise false.
+func inSlice(n string, h []string) bool {
+	for _, v := range h {
+		if v == n {
+			return true
+		}
+	}
+	return false
+}
+
+// convertTerraformSetToStringSlice turns a terraform set into a string slice.
+func convertTerraformSetToStringSlice(i interface{}) []string {
```

Comment:
the expandStringList() function already does this functionality so we actually don't need this function

 Also there doesn't seem to be any benefit in having an interface{} argument. Just accept `*schema.Set` since its easy to cast and you also make it explicitly clear by the name that it only accepts terraform sets.